.github/workflows/main-build.yml

```
@@ -237,6 +237,15 @@ jobs:
       appsignals-adot-image-name: ${{ needs.build.outputs.staging-image }}
       caller-workflow-name: 'main-build'
 
+  # AppSignals specific e2e tests for ec2
+  appsignals-e2e-ec2-test:
+    needs: build
+    uses: ./.github/workflows/appsignals-e2e-eks-test.yml
+    secrets: inherit
+    with:
+      aws-region: ${{ needs.build.outputs.aws_default_region }}
```

Why don't we use this for consistency?

```suggestion
      aws-region: ${{ needs.default-region-output.outputs.aws_default_region }}
```

---

Will also need dependency on the `default-region-output` step.